src/app/main/login/login.component.html

```diff
@@ -1,3 +1,4 @@
 <div class="login-page">
-    <dsp-login-form (loginSuccess)="refresh($event)"></dsp-login-form>
+    <dsp-login-form (loginSuccess)="refresh()"></dsp-login-form>
```

Comment:
The `LoginComponent` emits a boolean value in `loginSuccess`. I do not understand why you removed it?

Reply:
reversed in 27bc05c